scripts/_0_7_5_fix_observed_adms.py

```diff
@@ -0,0 +1,241 @@
+'''
+Ticket:
+https://nextcentury.atlassian.net/jira/software/projects/ITM/boards/116?assignee=60ba425da547eb00686ee0ce&jql=assignee%20%3D%2060ba425da547eb00686ee0ce&selectedIssue=ITM-1000
+'''
```

Comment:
Recommend simplifying to https://nextcentury.atlassian.net/browse/ITM-1000.  This is the URL you get when you click on the link icon (Copy Link) at the top of the ticket.